install.sh

```diff
@@ -269,9 +269,22 @@ check_path() {
 	if gum_func confirm "create /usr/local/bin/tea?" --affirmative="make symlink" --negative="skip"
 	then
 		echo  #spacer
-		sudo mkdir -p /usr/local/bin
-		sudo ln -sf "$tea" /usr/local/bin/tea
 
+		if [ -w /usr/local/bin ] || mkdir -p /usr/local/bin >/dev/null 2>&1;
+		then
+			ln -sf "$tea" /usr/local/bin/tea
+		elif which sudo >/dev/null;
+		then
+			sudo mkdir -p /usr/local/bin
+			sudo ln -sf "$tea" /usr/local/bin/tea
+		else
+			echo  #spacer
+			gum format -- <<-EOMD
```

Comment:
This line needs to be `gum_func` as of ff81688b1398760b3ea0ea8f03f4bc9193a028a2. Otherwise, I think we've gotten where we want to! Party!

Reply:
Done!

Reply:
Resolved conflicts!!
Following job was added to ci.yml
```
specific-version:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - uses: ./
        with:
          version: 0.13.8
      - run:
          test "$(tea --version)" = "tea 0.13.8"
```